lib/Share/RoomShareProvider.php

```diff
@@ -28,6 +28,7 @@
 
 namespace OCA\Talk\Share;
 
+use OC\Cache\CappedMemoryCache;
```

Comment:
```suggestion
use OCP\Cache\CappedMemoryCache;
```

Should use the public one